DocumentedExamples/Cross-contour_transport.ipynb

```diff
@@ -0,0 +1,1480 @@
wrHEFlJTnXZ03/piER9I32rBnV6jgyswuMLPHzez+YDu14rPDzGyxma0ysxVmNiT8cAEAAAAgnaLIXF3q7jOC7UZJMrNWST+R9BF3nybpTZJei+BeAAAAwIB9YvzcuroGUt6FgYjrfzdvlrTc3R+QJHd/yt07Y7oXGpDmduxdf3VEv593HkeKHDtt+ce5tQ8CAKCKyq6BiJd59FuaRRFcnWdmy83sSjMbFuybIsnN7GYzu9fMPt3XyWa2wMyWmdmy17Q9guEgr1hIGJVGfeNOAiwAQO5c//iypIcQLda56s7MbjWzlVW2eZIulzRZ0gxJWyR9IzitVdIbJP1t8PV0Mzuh2vXdfaG7z3L3WbtocATfEgAAAAA0X83gyt1PdPfpVbbr3P1Jd+909y5JV0iaHZy2SdLt7r7N3V+WdKOk18f3bQBSx0mzkh4CmqScsRr1jdo180AePfbzw5IeAoCYvH10jv49E0cb9jyXBZrZqIq3p0taGby+WdJhZrZ70NzijZIeDHMv5N+gP9w34HM7Tpql1luW6bU3h/8P0ivzjgp9DcSrMqiiNBAAAKRF2DlXFwdt1pdLOk7SJyXJ3Z+R9E1Jd0u6X9K97v7rkPcC+tR6S6k+eZffhg+whlx3VxRDQkye+IfuwRTZKxTR+DOWJz0EIPc+NO4NTb9n7uZbSYXLXLWGOdndz+rns5+o1I4dAAAAQAGlvbtf1Gjhj9yJInuF9Drgm70zVVs+RWkg0J/HfzEt6SEAqEOu5lsVFMEVcmmX3zaWVn/l7bO7v2feVaaMuoTSQBTHQJpZjH7HqhhGAiBKuSwJlApXFkhwhdxqJHs15Pql3d8z7ypzyF6hKJhvBTTPFRv+1LR7kbXKh1BzrgAAAACgTynPNEWNzBVyq9HSwJ7+chqlgVlCaSDQG3OtgHCS6BiIbCO4QqZ0njAz8mv2nG9VttuvKA0EkC615ltt+p/p3V6PfscqAiwAiTGPZ0szgivk2msnH1nzmJ7zrZBePde46ol5V8i7WvOtxrxzpaRSYFV+TTMLIJxmzLvKbTMLSXKLfksxgisAAAAAiAANLZApLbfd09Dxu9x8d6j7/eW0oygPzBDmXSFvHrumogzQ6usUWJm1AhBeM+Zddakr9nskJuVlfFEjuELmdZw4S623xpNOJ7BKl2oLCAN5Nv7/Nd52ncAKyJ7TRlef/43soSwQhfLqKbXnYPVE18Bs2fxPzLtCPqz/2eFJDwFA4MqNzVvvKm+K1tCCzBUAAACAeKQ8GIoamauCaX3dQUkPIXKNlATu+pvG52BRGpgtB/4bpYPIhwl/80DSQwAQeP9Y1rtCfQiuUDgDKg08ndLALKE0EFlHSSCQPpQGDgDrXAH5NZCgqmy3X5K9arYnPjlXT3xyZ5BUa42rSmSvkGXrf3Y4WSsghcheoR7MuQIAAAAQj5RnmqJGcIXM6TxhZkPrXb128pHa5ea7d8y3Gsi8K6lUGkgGq3kOuLSUfSpnr2jDjrxad/XOEkAz5loByBmCKyDdmr2QcBmBVTLKQRaQR2uvmqFJZ96f9DAAJOhXjy9lnascYc5VwXQ8sjrpIUSq48RZTb0fjS2a44m/D9+QgqYWyIJJ774/6SEASFjeAysaWgAAAAAAGkZwhUwqZ6zqXePqtZMH3imw0m6/vIvsVcye+Pu5OuBb4UsB6+kY+PhnyW4hOWuvmpH0EAAAESO4QmQGvX5a0+7VeuuyhkoCo5p3JZUCrJffQYAVh6gCq7L+SgM3f2auRn+N+VxIDiWBQPZEvdbVrx5fGun1kDyCK0Sm695VTb1fvVmrOOz+C5pbxCHKwErqO3u1+TNzdeDXCayQHLJWAKT8z7eSVOoWGPVWg5mNNbP/M7OHzGyVmX0i2D/czG4xs/bg67CKc843s9Vm9oiZnVyxf6aZrQg++7aZWX/3JrgCAAAAEL0YmlnU2dCiQ9I/uvshkuZIOtfMpkr6rKTb3L1N0m3BewWfnSlpmqRTJF1mZi3BtS6XtEBSW7Cd0t+NCa4QqWaWBiaN0sBoPRlBh8BqNn+693XJWiFplAQCQHzcfYu73xu8fkHSQ5JGS5onaVFw2CJJpwWv50m62t23u/s6SaslzTazUZKGuvtid3dJP6o4pyqCK0Ri0BFTJTW/NBD5sX/EJYFlB158544Aa/On52rzZ2hiAQAYmPePfUPSQ8ieBMoCK5nZBElHSLpL0v7uvkUqBWCSRgaHjZa0seK0TcG+0cHrnvv7RHCFqlqmvy7pIQCRKQdYB158J1krJI75VgAQ2ggzW1axLah2kJntKel/JP29uz/fz/WqzaPyfvb3qbW/D1FcnSsfUcv016lz5SNJDwWIxIEXE1QhHSgJBPLpp5t2/v/Me8ZQJbFDPIv+bnP3fttGm9kuKgVW/+3uvwh2P2lmo9x9S1DytzXYv0nS2IrTx0jaHOwfU2V/n8hcAQAAAIicKZmGFkFHv/+S9JC7f7Pio+slzQ9ez5d0XcX+M81ssJlNVKlxxdKgdPAFM5sTXPPsinOqIrgqoNaD2+o6rihZq+1vG1gbVNqxA2jU2p8ekfQQAMTkPWPm7tiu3rQ46eEU3TGSzpJ0vJndH2ynSvqapJPMrF3SScF7ufsqSddIelDSbySd6+6dwbXOkfR9lZpcrJF0U383piywgDoebk96CABQSJPec1/SQwAQgx9vvENnjT1mx/szxxytqzct1pljjk5wVCkRT1lg/7d0/5Oqz5eSpBP6OOciSRdV2b9M0vR6703mqqDqzV4VweD/ZXV0APEjawXkV2VgVUZgVUxkrgAAAABEr/5Ff3ODzFVBdTzcTvYKqRHXAsJAmlASCGTfoo139Nr34yr7yph7pcTXuWo2gqsCqyfAYr0rNENcCwgDaUFJIJBf1UoCy8pzr1AcBFcFR3OLgXn5HUclPQQAGULWCiiuM8ccrZ9vWpL0MJJD5goAAAAA0CgaWqBfRVnrqh4vv3OOdv+f0l+eWOMKQL3W/vQ
```

Comment:
```suggestion
    "days_in_month = ssh.time.dt.days_in_month\n",
    "days_in_year = 365\n",
    "ty_trans = (ty_trans * days_in_month / days_in_year).sum('time')\n",
    "tx_trans = (tx_trans * days_in_month / days_in_year).sum('time')\n",
```

Reply:
I stopped looking at the notebook line-by-line at this point. But as a general remark:
- commas after spaces (both in code and in English)
- make the code as verbose as possible (e.g., by using variable names that are self-explanatory)